### PR TITLE
Fix for issues #1007 and #1008, additional bug fixes for Aircrack-ng 

### PR DESCRIPTION
**Pull request by bshean1**
Due to trimester length and the overlaps from the bugs, multiple changes have been made.

Fixed GitHub issues #1007 and #1008.
Gave WEP mode the ability to specify packet capture file.
Removed wordlist option from WEP mode as it is not needed.
Removed required from arguments that are not required for the tool to run.
Removed unused and unnecessary placeholders.
Set selectedModeOption to default to WEP to make sure WEP specific options are loaded in the first time the tool is opened.
Commented out securityType options, as it is made redundant by Aircrack-ng performing that automatically.
Added code to reset advanced mode back to default when option is deselected.

Rewrote construct arguments section as mode specific options were not working and to improve code readability.
Nested if statements were and are still not working and the reason is unclear.
A temporary fix was made to reset the form every time the mode is switched.
Code for mode specific options was left in for a future fix.

Other notes:
There is an option for customConfig that was present in the code, however it has not been implemented.

### CODE REVIEW

File: src/components/AircrackNG/AircrackNG.tsx [hunk 1]
```diff
@@ -35,8 +35,8 @@ const description = "Aircrack-ng is a tool for cracking WEP and WPA/WPA2 passphr
 const steps =
     "How to use Aircrack-ng:\n\n" +
     "Step 1: Type in the path to wordlist(s) filename(s) including the extension .txt. e.g. 'example.txt'.\n" +
-    "For files containing hexadecimal values, you must put a “h:” in front of the file name.\n" +
-    "Step 2: Select the target network based on the access point's MAC address.\n" +
+    "For files containing hexadecimal values, you must put a “h:” in front of the file name.\n";
```
Comment (cjboyd1999) [inline src/components/AircrackNG/AircrackNG.tsx:38]: Looks like the edit made to this line, changing `+` to `;` breaks the displayed steps in the user guide:
image

The semicolon (;) terminates the assignment of the `steps` variable, so the remaining steps are not included in this assignment. 

Can you change this back to a `+` to include the remaining steps?

Comment (cjboyd1999) [reply src/components/AircrackNG/AircrackNG.tsx:38]: Noting also that on the following line where step 2 is written, you'll need to remove the leading `+` to follow the format of the other sentences.

**Review by cjboyd1999 — CHANGES_REQUESTED** [review]
As noted in my comments below, please edit the code regarding the User Guide's steps to fix the issue with steps 2+ not being displayed.

File: src/components/AircrackNG/AircrackNG.tsx (hunk 1 above)
Comment (bshean1) [reply src/components/AircrackNG/AircrackNG.tsx:38]: Yeah, this wasn't something that I edited, running npx prettier made this change and I'm not sure why but I should have checked it after.

Comment (bshean1) [reply src/components/AircrackNG/AircrackNG.tsx]: It should be fixed now.

**Review by cjboyd1999 — APPROVED** [review]
Changes look good, no issues when testing locally. Approved for merging.

**Pull request merged**